tools/local_llm_bakeoff.py

```
@@ -49,6 +50,27 @@ def _artifact_name(profile_id: str, suffix: str) -> str:
     return f"{safe}.{suffix}"
 
 
+def resolve_github_token() -> str:
+    token = os.environ.get("GITHUB_TOKEN")
+    if token:
+        return token
+    try:
+        completed = subprocess.run(
+            ["gh", "auth", "token"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)